libs/cardano-ledger-core/test/Test/Cardano/Ledger/AddressSpec.hs

```diff
@@ -145,3 +143,17 @@ propDeserializeRewardAcntErrors v acnt = do
       ("Mingled address with " ++ mingler ++ " was parsed: " ++ show badAddr)
     $ isNothing
     $ CA.decodeRewardAcnt @c badAddr
+
+spec :: Spec
+spec = do
+  describe "CompactAddr" $ do
+    prop "compactAddr/decompactAddr round trip" $
+      propCompactAddrRoundTrip @StandardCrypto
+    prop "Compact address binary representation" $
+      propCompactSerializationAgree @StandardCrypto
+    prop "Ensure Addr failures on incorrect binary data" $
+      propDecompactErrors @StandardCrypto
+    prop "Ensure RewardAcnt failures on incorrect binary data" $
+      propDeserializeRewardAcntErrors @StandardCrypto
+    prop "Decompacting an address is still valid" $
+      propValidateNewDecompact @StandardCrypto
```

Comment:
Three of these were part of the `minimalPropertyTests`  in addition to the `propertyTests`, so I guess the restructuring in of tests in `cardano-ledger-core` has an impact on what tests the "nightly builds" are running. This is just an observation, because I don't even know if  the nightly builds are running at all, or if it matters. Just noticing the difference.

Reply:
Moving those tests to hspec will run them regardless of where the test suite is executed, which is what we want. 

I am not sure what was the reasoning behind running some tests on nightly only, but I think we need to move away from that. All tests should run at all times. One reason I can think of is runtime. If some test takes long time, then that test should be fixed. Rather than making it run once a day only.

In any case, this was a good observation, but this minimal vs normal vs nightly distinction is not important for this PR.

Reply:
Fair enough